src/hades_types/common.py

```
@@ -6,7 +6,8 @@
 of truth for all common types, enums, and type aliases.
 """
 
-from typing import Dict, List, Any, Optional, Union, TypedDict, NewType, ForwardRef, TypeAlias
+from typing import Dict, List, Any, Optional, Union, TypedDict, NewType, ForwardRef
```

Ruff (0.11.9)

`typing.ForwardRef` imported but unused

Remove unused import: `typing.ForwardRef`

(F401)